django/contrib/postgres/fields/ranges.py

```diff
@@ -132,6 +133,44 @@ def db_type(self, connection):
 RangeField.register_lookup(lookups.Overlap)
 
 
+class DateTimeRangeContains(models.Lookup):
```

Comment:
What about _Specialized lookup for Date/DateTimeRange containment that explicitly type cast its rhs to its rhs type to work around a limitation of Psycopg2/PostgreSQL_?

FWIW it seems to be caused by the same limitation PR #7099 tried to fix for [ticket #24726](https://code.djangoproject.com/ticket/24726).